monasca/templates/notification-deployment.yaml

```diff
@@ -91,3 +91,22 @@ spec:
             - name: NF_SLACK_PROXY
               value: {{ .Values.notification.plugin_config.slack.proxy | quote }}
             {{- end }}
+            {{- if .Values.notification.plugin_config.hipchat.template }}
+            - name: NF_HIPCHAT_TEMPLATE
+              value: "/hipchat-template.yml.j2"
+            {{- end }}
+          {{- if .Values.notification.plugin_config.hipchat.template }}
+          volumeMounts:
+            - name: hipchat-template
+              mountPath: /hipchat-template.yml.j2
+              subPath: hipchat-template.yml.j2
+          {{- end }}
```

Comment:
it felt wrong to let an if span over multiple levels of (de-)indentation, it can be changed easily enough